### PR TITLE
MTV-1096: Release Notes for MTV 2.7.0

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1096](https://issues.redhat.com/browse/MTV-1096)

### PREVIEW

* [Release Notes MTV 2.7.0](https://anarnold97.github.io/MTA-Preview/MTV/rn-27.html)

### CODE REVIEW

File: documentation/modules/rn-2.7.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+
```
Comment (anarnold97) [inline documentation/modules/rn-2.7.adoc:18]:
```suggestion
```

File: documentation/modules/new-features-and-enhancements-2-7.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| |Duration |VM downtime
+
+|Cold migration
+|Minimal
+|High
+
+|Warm migration
+|Longer and variable
+|Minimal
+|===
```
Comment (anarnold97) [inline documentation/modules/new-features-and-enhancements-2-7.adoc:27]:
```suggestion
```

File: documentation/modules/new-features-and-enhancements-2-7.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+[id="new-features-and-enhancements-2-7_{context}"]
+= New features and enhancements
+
+{project-first} 2.7 introduces the following features and enhancements:
+
+
+[id="new-features-and-enhancements-2-7-0_{context}"]
+== New features and enhancements 2.7.0
+
+* In {project-short} 2.7.0, warm migration has been enhanced.
```
Comment (fabiand) [inline documentation/modules/new-features-and-enhancements-2-7.adoc:11]: Warm migration is now based on RHEL 9 inheriting features and bug fixes.

File: documentation/modules/rn-2.7.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-27_{context}"]
+= {project-full} 2.7
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
```
Comment (fabiand) [inline documentation/modules/rn-2.7.adoc:10]: VMware vSphere 6, 7, and 8

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+In earlier releases of {project-short}, a status condition indicating a failed mapping resource of a plan was not added to the plan. This issue has been resolved in {project-short} 2.7.0, with a status condition indicating the failed mapping being added. link:https://issues.redhat.com/browse/MTV-1461[(MTV-1461)]
+
+.ifcfg files with HWaddr cause the NIC name to change
+
+In earlier releases of {project-short}, interface configuration (ifcfg) files with a hardware address (HWaddr) of the Ethernet interface, caused the name of the network interface controller (NIC) to change. This issue has been resolved in {project-short} 2.7.0. link:https://issues.redhat.com/browse/MTV-1463[(MTV-1463)]
+
+.Import fails with special characters in VMX file
+
+In earlier releases of {project-short}, imports failed when there were special characters in the parameters of the VMX file. This issue has been resolved in {project-short} 2.7.0. link:https://issues.redhat.com/browse/MTV-1472[(MTV-1472)]
+Import fails with special characters in VMX file
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:28]:
```suggestion
```

**Review by fabiand — APPROVED** [review]

**Pull request merged**